pyinfra/api/util.py

```diff
@@ -222,6 +223,7 @@ def get_template(filename_or_string, is_string=False):
     TEMPLATES[cache_key] = Environment(
         undefined=StrictUndefined,
         keep_trailing_newline=True,
+        loader=FileSystemLoader(getcwd()),
```

Comment:
This should _probably_ be using the deploy directory, but pending #698 I'm happy with this.

(I hate the deploy dir stuff it's SUPER confusing).